python/friesian/DEVCATALOG_toolkit.md

```diff
@@ -196,15 +198,23 @@ python train_2tower.py \
     --data_dir ../../../../apps/wide-deep-recommendation/recsys_data/preprocessed \
     --model_dir recsys_2tower/ \
     --batch_size 8000
+
+python predict_2tower.py \
+    --backend spark \
+    --executor_cores 8 \
+    --executor_memory 6g \
+    --data_dir ../../../../apps/wide-deep-recommendation/recsys_data/preprocessed \
+    --model_dir recsys_2tower/
 ```
 
 **Expected Training Workflow Output**
 
 Check out the processed data and saved models after the training:
 ```
+cd /path/to/BigDL
 ll apps/wide-deep-recommendation/recsys_data/preprocessed
 ll recsys_wnd/
```

Comment:
??

Reply:
user might in another directory when installing redis